### ISSUE TITLE
time to end sensor stuck at 1min

### ISSUE THREAD

**Issue by jo-me** [albaintor/homeassistant_electrolux_status#116]
Hi,

I read somewhere that issue with the stuck time-to-end sensor has been resolved internally in this component.
Is this already supposed to work in the latest version?

If so, then it does not work in my case (AEG 9000 Series). 
It still gets stuck at 1min forever.

Is there any hope?

Thanks
Jochen

**Comment by Palmdale95** [comment]
Hey there. I don't know if this is still valid but I experieced just today that the time_to end ist not even moving. Looks like it is not refreshing time by time. So it just gets stuck at the beginning of start program. 

From my side I have everything up to latest stable versions (core, HA, integration...)

Is somebody experiencing this issue , too?

**Comment by Joseminglein** [comment]
Hi @Palmdale95 , yes,  I see that issue also, it seems that sensor is not up to date (although sometimes it is), meanwhile the app provides the exact value

**Comment by Palmdale95** [comment]
Ok then we have an issue here. Looks like it is API/integration relatet.  @albaintor mind taking a look or maybe you are already on it?

From my side I try to cover it with an outomation that reloads the integration as long as the maschine is running to update the data an asure the other related automations are working correctly. It is not smooth but I hope it covers the topic the time beeing.

**Comment by Palmdale95** [comment]
Correct. tha t how the End issue was solved the last time. But right now it is related a little different because the sensor ist not updating at all or just sometimes, when the machine is running. It is really strange. Just recognised it because it stated 7 minutes until the end (example) and I was wondering why I did not get message when finished after 20 minutes. Had a look if my integration was faulty but the sow the sensor did not upgrade anymore. Same thing the happened with next machine running. Just stated the start and then that's it. Maybe the update intervals has changed and now it is to long to fetch the end? Just an assumption.
